Source/Core/VideoCommon/OnScreenUI.cpp

```diff
@@ -73,11 +75,28 @@ bool OnScreenUI::Initialize(u32 width, u32 height, float scale)
   }
 
   // Font defaults
+  ImGuiIO& io = ImGui::GetIO();
   m_imgui_textures.clear();
 
-  // Setup new font management behavior.
-  ImGui::GetIO().BackendFlags |=
-      ImGuiBackendFlags_RendererHasTextures | ImGuiBackendFlags_RendererHasVtxOffset;
+  // User supplied font
+  std::string file = File::GetUserPath(D_LOAD_IDX) + "OSD_Font.ttf";
```

Comment:
It will be non-obvious to users that they can override the VeraMono.ttf file by providing a file in the User folder with a different name. What do you think about giving it the name OSD_Font.ttf in the Sys folder too?

Reply:
>  I could add a description to OSD Font Size, to mention loading a custom font.

That's probably a good idea.

Reply:
```
<br><br>The font can be changed by placing a TTF font file into Dolphin's "
"User/Load folder, and renaming it OSD_Font.ttf." 
```
I didn't rename the file in System, let me know if you still want me to do that.